src/Symfony/Component/HttpFoundation/RequestStack.php

```diff
@@ -47,7 +47,7 @@ public function push(Request $request)
     public function pop()
     {
         if (!$this->requests) {
-            return;
+            return null;
```

Comment:
Fixing inconsistent return points should be done on a lower branch and is out of scope for this PR.